spec/models/medicaid_application_attributes_spec.rb

```diff
@@ -21,7 +21,29 @@
 
     describe "insurance types" do
       it_should_behave_like "insurance type", "medicaid", "Medicaid"
-      it_should_behave_like "insurance type", "medicare", "Medicare"
+
+      context "applicant is on medicare" do
```

Comment:
shoud we add "and needs help paying medical bills from the past 3 months" to this  context?

Reply:
That feels unrelated. This is the box we're checking.

![pasted image at 2017_11_21_06_03 pm](https://user-images.githubusercontent.com/2306794/33290354-4588b16a-d377-11e7-9243-936b78700301.png)

Reply:
gotcha, I interpreted this feature to be related to that question but it sounds like it isn't

Reply:
Yee